src/vue/pages/Limits/Limits.RequestDetailsViewer.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <div class="limits-request-details-viewer">
+    <div
+      v-if="request.isApproved"
+      class="request-state request-state--approved"
+    >
+      <p class="request-state__content">
+        {{ 'limits-request-details-viewer.approved-msg' | globalize }}
+      </p>
+    </div>
+    <div
+      v-else-if="request.isPending"
+      class="request-state request-state--pending"
+    >
+      <p class="request-state__content">
+        {{ 'limits-request-details-viewer.pending-msg' | globalize }}
+      </p>
+    </div>
+
+    <div
+      v-else-if="request.isRejected"
+      class="request-state request-state--rejected"
+    >
+      <p class="request-state__content">
+        <!-- eslint-disable-next-line max-len -->
+        {{ 'limits-request-details-viewer.rejected-msg' | globalize() }}
+      </p>
+    </div>
+
+    <div
+      v-else-if="request.isCanceled"
+      class="request-state request-state--canceled"
+    >
+      <p class="request-state__content">
+        {{ 'request-messages.canceled-msg' | globalize() }}
+      </p>
+    </div>
+
+    <div
+      v-else-if="request.isPermanentlyRejected"
+      class="request-state request-state--permanently-rejected"
+    >
+      <p class="request-state__content">
+        <!-- eslint-disable-next-line max-len -->
+        {{ 'request-messages.permanently-rejected-msg' | globalize({ reason: request.rejectReason }) }}
+      </p>
+    </div>
+    <div class="app__table limits-request-details__table">
```

Comment:
`limits-request-details__table` => `limits-request-details-viewer__table`

pls recheck that all the other classes are compliant with BEM

Reply:
Fixed